packages/nextjs/components/scaffold-eth/Contract/WriteOnlyFunctionForm.tsx

```diff
@@ -123,24 +140,31 @@ export const WriteOnlyFunctionForm = ({
               {displayedTxResult ? <TxReceipt txResult={displayedTxResult} /> : null}
             </div>
           )}
-          {connectedAddress ? (
-            <div
-              className={`flex ${
-                wrongNetwork &&
-                "tooltip before:content-[attr(data-tip)] before:right-[-10px] before:left-auto before:transform-none"
-              }`}
-              data-tip={`${wrongNetwork && "Wrong network"}`}
-            >
-              <button className="btn btn-secondary btn-sm" disabled={wrongNetwork || isPending} onClick={handleWrite}>
-                {isPending && <span className="loading loading-spinner loading-xs"></span>}
-                Send 💸
+          <div className="flex gap-2">
+            <div className="tooltip tooltip-left" data-tip="Copy Calldata">
+              <button className="btn btn-ghost btn-sm" onClick={handleCopyCalldata}>
+                <DocumentDuplicateIcon className="h-5 w-5" />
               </button>
             </div>
```

Comment:
Can you check how `Address` component show this icon it uses ReactCopyToClipboard wrapper along with icon change when clicked? We make the ux similar to it: 


https://github.com/user-attachments/assets/0aeb0b6c-a63a-4fcc-9557-f9534001efb6